galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/RequestorRas.java

```diff
@@ -106,13 +99,17 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
        
     }
     
-    private List<String> getRequestors(HttpServletResponse resp) {
+    private List<String> getRequestors(HttpServletResponse resp) throws ResultArchiveStoreException{
     	
+    	List<String> requestors = new ArrayList<>();
+    			
+    	for (IResultArchiveStoreDirectoryService directoryService : framework.getResultArchiveStore().getDirectoryServices()) {
+    		requestors = directoryService.getRequestors();
```

Comment:
you need to AddAll here or the second directoryservice call will overright the first one